src/app/Http/Controllers/Query/SwitcherController.php

```diff
@@ -536,8 +536,12 @@ private function getServerInfo($request): BaseJsonResponse
             'emailEnabled' => legacy_config('enable_email_contact') ? '1' : '0',
             'emailIncludesServiceBodies' => legacy_config('include_service_body_admin_on_emails') ? '1' : '0',
             'changesPerMeeting' => strval(legacy_config('change_depth_for_meetings')),
-            'meeting_states_and_provinces' => implode(',', legacy_config('meeting_states_and_provinces', [])),
-            'meeting_counties_and_sub_provinces' => implode(',', legacy_config('meeting_counties_and_sub_provinces', [])),
+            'meeting_states_and_provinces' => is_array(legacy_config('meeting_states_and_provinces', []))
```

Comment:
we have to do opposite basically here because ServerInfo outputs as comma separated string